plugins/emoticons.md

```diff
@@ -84,13 +84,13 @@ This option provides the default location to load the emoji database from. The d
 **Default:** `${pluginUrl}/js/emojis.js`
 
 ```js
-window.tinymce.emoticons_plugin_database = {
+tinymce.Resource.add('tinymce.plugins.emoticons', {
   robot: {
     keywords: ['computer', 'machine', 'bot'],
     char: '🤖',
     category: 'people'
   }
-}
+})
```

Comment:
There should be a semicolon at the end of the line here.

Reply:
fixed by https://github.com/tinymce/tinymce-docs/pull/1532/commits/bbc40692a57ef6ef3deb259559d360f4d3b5abaa